config/vufind/config.ini

```diff
@@ -1915,7 +1915,7 @@ ja          = "Japanese"
 nl          = "Dutch"
 ;nl-be       = "Flemish Dutch"
 pt          = "Portuguese"
-pt-br       = "Brazilian Portugese"
+pt-br       = "Brazilian Portuguese"
```

Comment:
This value is used as a key in native.ini, so it is never shown to the end user -- it gets translated into the native name for the language. If we fix it here, we also need to fix the key in native.ini. However, fixing it may cause more problems than it is worth, since it will break customizations that have been made using the legacy key, and it will also prevent translation from working correctly if anyone has an outdated config.ini file in the future. A compromise might be to add a translation to native.ini for the correctly spelled version and retain the one for the misspelled one.